core/src/main/resources/hudson/model/View/newJobButtonBar.jelly

```diff
@@ -23,6 +23,6 @@ THE SOFTWARE.
 -->
 
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
```

Comment:
unrelated, but looks fine

Reply:
Ok than. We can keep them, or use some tools to clean all of them.